packages/hoppscotch-common/src/components/http/SaveResponseName.vue

```diff
@@ -48,22 +56,49 @@ const props = withDefaults(
   defineProps<{
     show: boolean
     loadingState?: boolean
-    modelValue?: string
+    responseName?: string
+    hasSameNameResponse?: boolean
   }>(),
   {
     show: false,
     loadingState: false,
-    modelValue: "",
+    responseName: "",
+    hasSameNameResponse: false,
   }
 )
 
 const emit = defineEmits<{
   (e: "submit", name: string): void
   (e: "hide-modal"): void
-  (e: "update:modelValue", value: string): void
+  (e: "update:responseName", value: string): void
+  (e: "update:hasSameNameResponse", value: boolean): void
 }>()
 
-const editingName = useVModel(props, "modelValue")
+const editingName = useVModel(props, "responseName")
+
+const hasSameNameInspectionResult: Ref<InspectorResult[]> = computed(() => {
```

Comment:
instead of using `Ref<InspectorResult[]>` as type, might be better to go with `ComputedRef<InspectorResult[]>`, using `ComputedRef` will give us type errors if we're trying to write to it, where `Ref` won't. Since the code is using `computed` it will throw at runtime.